src/functionalTest/resources/data/family-daily-cause-list/familyDailyCauseList.json

```diff
@@ -51,6 +51,7 @@
                             "caseNumber": "12341234",
                             "caseJurisdictionET": "Family Law",
                             "caseSequenceIndicator": "2 of 3",
+                            "caseType": "Case type",
```

Comment:
Hey Alex, the test file needs updated for this should be in civil-and-family-daily-cause-list